### PR TITLE
Added some log commands in the servlet

### PR DESCRIPTION
**Pull request by hibby96**
Just filled in a bunch of log commands.  Shouldn't need to do anymore.

### CODE REVIEW

File: JenkinsService/src/main/java/com/revature/JenkinsServlet.java [hunk 1]
```diff
@@ -86,10 +100,12 @@ void makeJob() throws IOException {
         try {
             exitCode = process.waitFor();
         } catch (InterruptedException e) {
+            log.error("Exception occurred within makeJob method.  " + e.getMessage());
             throw new IOException(e.getMessage());
         }
 
         if (exitCode != 0) {
+            log.error("makeJob method failing due to cURL failure, exitCode=0");
```
Comment (ItsAlxl) [inline JenkinsService/src/main/java/com/revature/JenkinsServlet.java]: The exitCode is _not_ 0, which is the indication that it failed

**Review by ItsAlxl — CHANGES_REQUESTED** [review]
Fix line 108; the log claims exitCode=0 when exitCode strictly isn't 0

File: JenkinsService/src/main/java/com/revature/JenkinsServlet.java (hunk 1 above)
Comment (hibby96) [reply JenkinsService/src/main/java/com/revature/JenkinsServlet.java]: corrected

**Review by ItsAlxl — APPROVED** [review]
👍

**Pull request merged**